tests/tablenavigation.js

```diff
@@ -1714,6 +1714,8 @@ describe( 'TableNavigation', () => {
 						`
 					) );
 					global.document.querySelector( 'head' ).appendChild( styleElement );
+
+					editor.editing.view.focus();
```

Comment:
Could you add comment about this. I reckon that this cause problems but I don't remember what was exactly problematic here. Anyway maybe I'll learn why we do that :D

Reply:
Addressed in af0b27f.